### PR TITLE
[JENKINS-308] Set Emulator Start Log to verbose

### PR DESCRIPTION
**Pull request by dermooods**
added verbose flag to emulator start

### CODE REVIEW

File: android-emulators-gradle/src/main/groovy/org/catrobat/gradle/androidemulators/AvdCreator.groovy [hunk 1]
```diff
@@ -44,7 +44,7 @@ class AvdCreator {
         avdStore.store(avdName, settings) {
             def avdmanager = new CommandBuilder(Utils.joinPaths(sdkDirectory, 'tools', 'bin', 'avdmanager'), '.bat')
 
-            avdmanager.addArguments(['create', 'avd', '-f', '-n', avdName])
+            avdmanager.addArguments(['create', 'avd', '-f', '-verbose', '-n', avdName])
```
Comment (mfuchs) [inline android-emulators-gradle/src/main/groovy/org/catrobat/gradle/androidemulators/AvdCreator.groovy]: That did not work for me.
It mentions that there is no "-verbose" flag,  instead there is "-v" and "--verbose".
Yet neither of these worked and I got "Error: Flag '--verbose' is not valid for 'create avd'."

Did this work for you?
Maybe we have different versions of the Android SDK.

**Review by mfuchs — CHANGES_REQUESTED** [review]

**Comment by dermooods** [comment]
Hi, no we decided to not merge this pull request, I corrected it an will test it today.
Thanks
 

Gesendet: Freitag, 19. Juli 2019 um 09:03 Uhr
Von: "Matthias Fuchs" <notifications@github.com>
An: Catrobat/Gradle <Gradle@noreply.github.com>
Cc: dermooods <dermutz@gmx.at>, Author <author@noreply.github.com>
Betreff: Re: [Catrobat/Gradle] [JENKINS-308] Set Emulator Start Log to verbose (#7)


@mfuchs requested changes on this pull request.


In android-emulators-gradle/src/main/groovy/org/catrobat/gradle/androidemulators/AvdCreator.groovy:

> @@ -44,7 +44,7 @@ class AvdCreator {
         avdStore.store(avdName, settings) {
             def avdmanager = new CommandBuilder(Utils.joinPaths(sdkDirectory, 'tools', 'bin', 'avdmanager'), '.bat')

-            avdmanager.addArguments(['create', 'avd', '-f', '-n', avdName])
+            avdmanager.addArguments(['create', 'avd', '-f', '-verbose', '-n', avdName])


That did not work for me.
It mentions that there is no "-verbose" flag, instead there is "-v" and "--verbose".
Yet neither of these worked and I got "Error: Flag '--verbose' is not valid for 'create avd'."

Did this work for you?
Maybe we have different versions of the Android SDK.

File: android-emulators-gradle/src/main/groovy/org/catrobat/gradle/androidemulators/AvdCreator.groovy (hunk 1 above)
Comment (dermooods) [reply android-emulators-gradle/src/main/groovy/org/catrobat/gradle/androidemulators/AvdCreator.groovy]: Hi, no we decided to not merge this pull request, I corrected it an will test it today.
Thanks

**Comment by jampo** [comment]
Done with PR JENKINS-308

**Pull request closed without merging**